src/Carnac.Tests/MessageProviderFacts.cs

```diff
@@ -61,12 +67,12 @@ public void recognises_shortcuts()
         }
 
         [Fact]
-        public void does_not_show_shortcut_name_on_partial_match()
+        public void does_not_show_key_press_on_partial_match()
```

Comment:
I'll allow this, but I'd also love to wrap my head around the new chord changes and how it feels in the various apps (not just VS) so I might come back to this with better feedback.